### PR TITLE
feat: Allow to configure and deploy RDMA CNI image

### PR DESCRIPTION
**Pull request by e0ne**
Allow to configure and deploy RDMA CNI image

### CODE REVIEW

File: hack/release.yaml [hunk 1]
```diff
@@ -73,3 +73,7 @@ ovsCni:
   image: ovs-cni-plugin
   repository: nvcr.io/nvstaging/mellanox
   version: 61fd27c
+rdmaCni:
+  image: rdma-cni
+  repository: ghcr.io/k8snetworkplumbingwg
+  version: latest
```
Comment (rollandf) [inline hack/release.yaml:79]: I think we should have a specific version here, no?

Comment (e0ne) [reply hack/release.yaml:79]: I would like to keep `latest` for beta releases to simplify testing as it contains multi-arch images. If you think it's better to use some tag for beta release I'll update PR

Comment (rollandf) [reply hack/release.yaml:79]: Ack.
Just let's make sure that we do not release with `latest`

**Review by rollandf — APPROVED** [review]

**Review by ykulazhenkov — DISMISSED** [review]

**Pull request merged**